.github/workflows/cherry-pick-next-to-master.yml

```
@@ -0,0 +1,29 @@
+on:
+  pull_request:
+    branches:
+      - next
+    types: ['closed']
+
+jobs:
+  cherry_pick_to_master:
+    runs-on: ubuntu-latest
+    name: Cherry pick into master
+    permissions:
+      pull-requests: write
+      contents: write
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'needs cherry-pick') && github.event.pull_request.merged == true }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Cherry pick and create the new PR
+        uses: carloscastrojumo/github-cherry-pick-action@v1.0.9
+        with:
+          branch: master
```

Have you tested what name the cherry-pick PR gets? 🤔 
Maybe we need to add:
https://github.com/carloscastrojumo/github-cherry-pick-action?tab=readme-ov-file#action-inputs
```suggestion
          branch: master
          title: ${{old_title}}
```

---

Oh, thanks for confirming. 👍 
Sorry, I misunderstood their listed default. 🤷 🙈